### PR TITLE
docs(ngClass): add class 'has-error' to demonstrate hyphen use

### PR DESCRIPTION
**Pull request by qbzzt**
Sorry, I don't know how to put it in the original pull request. I'm new to GitHub. But here it is, with just one additional class, red-error, which is the only one with quotes.

   Thanks, 
        Ori

### CODE REVIEW

File: src/ng/directive/ngClass.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ function classDirective(name, selector) {
  * @example Example that demonstrates basic bindings via ngClass directive.
    <example>
      <file name="index.html">
-       <p ng-class="{strike: deleted, bold: important, red: error}">Map Syntax Example</p>
+       <p ng-class="{strike: deleted, bold: important, 'red-error': error}">Map Syntax Example</p>
```
Comment (gkalpak) [inline src/ng/directive/ngClass.js]: This is too similar to `red` (and not very intuitive) :)
How about changing it to `'has-error': error`.

Then you need to change the CSS class (to `has-error`) and also update the text on [line 176][1] to `error (apply "has-error" class)`.

[1]: https://github.com/angular/angular.js/pull/12027/files#diff-d0df966e85933ac85b0f511aa14ca126R176

**Comment by gkalpak** [comment]
@qbzzt, I left an inline comment. You can update this PR by simply pushing a new commit to your branch (the PR will be updated automatically).

<sub>BTW, you can find usefull info in [CONTRIBUTING.md][1] (in case you haven't already).</sub>

[1]: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md

**Comment by qbzzt** [comment]
Thank you. I did the changes in the comment.

**Comment by gkalpak** [comment]
@qbzzt, that was quick :) We are almost there. Now we have some failing e2e tests we need to fix.
<sub>(It's always a good idea to run `grunt test` before you push a new commit/PR, if you want to be sure all tests pass.)</sub>

Anyway, you need to change `/red/` to `/has-error/` on [line 219][1] and [line 225][2], because we are testing against the previous `red` class.

[1]: https://github.com/angular/angular.js/pull/12027/files#diff-d0df966e85933ac85b0f511aa14ca126R219
[2]: https://github.com/angular/angular.js/pull/12027/files#diff-d0df966e85933ac85b0f511aa14ca126R225

**Comment by qbzzt** [comment]
Sorry, I didn't know about the tests. Anyway, they're running now but with any luck they'll succeed.

**Comment by gkalpak** [comment]
Yup, they passed. (No luck, just deep knowledge and hard work :P)

Thx for working on this, @qbzzt !

It LGTM. Now all we have to do is sit back and wait (hopefully it won't take long) :)

**Comment by qbzzt** [comment]
It still hasn't merged. Is there a problem, or does it normally take a few days?

**Comment by gkalpak** [comment]
Yes, I meant _"wait for someone from the team to give it a LGTM"_ and **then** it can get merged.

**Comment by Narretz** [comment]
LGTM! 

For (simple) doc fixes you don't have to wait for another LGTM (unless you want to ;)

**Comment by gkalpak** [comment]
Merged

**Comment by qbzzt** [comment]
Thank you.

**Pull request closed without merging**